frontend/javascripts/oxalis/model_initialization.js

```diff
@@ -472,16 +472,13 @@ function setupLayerForVolumeTracing(
     // remember the name of the original layer, used to request mappings
     fallbackLayer: tracing.fallbackLayer,
     fallbackLayerInfo: fallbackLayer,
+    isTracingLayer: true,
   };
 
   if (fallbackLayer != null) {
     // Replace the original tracing layer
     layers[fallbackLayerIndex] = tracingLayer;
   } else {
-    // Remove other segmentation layers, since we are adding a new one.
-    // This is a temporary workaround. In the long term we want to support
-    // multiple segmentation layers.
-    layers = layers.filter(layer => layer.category !== "segmentation");
```

Comment:
for volume annotations, we could still kick out the other segmentation layer if there shouldn't be a fallback. might be less confusing, but I'm not sure about this.

Reply:
Yeah, this is a good question. What's cool about the current state in this PR is that nothing "gets lost". If a dataset already has a segmentation, the user can always toggle it back on. However, if the user said "start with a fresh segmentation layer", that can be confusing. A trade-off might be to disable the original segmentation layer when opening the annotation. However, this would need to be done by the back-end, as the front-end cannot really distinguish between the first time and the n+1-th time a user opens an annotation.

@fm3 What do you about this? Is it feasible, to adapt the `/api/dataSetConfigurations` route so that the volume layer X is enabled and all other segmentation layers are disabled if the route is requested for the first time with a volume tracing id X?

Reply:
I discussed this with @fm3 and we agreed on kicking out the existing segmentation layer if the user chooses "without fallback". The main problem is that the visibility of the segmentation layer is stored per dataset. This means, if I create an annotation without fallback for a dataset, the "old" segmentation layer is disabled automatically and opening the dataset in view mode will be affected by that. Then, the segmentation layer will be invisible there, which is most likely not what the user wants nor expects.

I'll add this part to the follow-up issue.